src/coreclr/jit/gentree.cpp

```diff
@@ -11836,15 +11834,7 @@ void Compiler::gtDispTree(GenTree*     tree,
             {
                 // old style intrinsic
                 assert(intrinsic->gtIntrinsicName == NI_Illegal);
-                switch (intrinsic->gtIntrinsicId)
-                {
-                    case CORINFO_INTRINSIC_Object_GetType:
-                        printf(" objGetType");
-                        break;
-
-                    default:
-                        unreached();
-                }
+                unreached();
```

Comment:
This should either  `unreached()` or `assert(intrinsic->gtIntrinsicName == NI_Illegal)` ?